types/node/index.d.ts

```diff
@@ -797,7 +797,7 @@ declare module "querystring" {
     }
 
     export function stringify<T>(obj: T, sep?: string, eq?: string, options?: StringifyOptions): string;
-    export function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): any;
+    export function parse(str: string, sep?: string, eq?: string, options?: ParseOptions): { [key: string]: string };
```

Comment:
I think the return type should be `{ [key: string]: string | string[] }` as `'foo=bar&abc=xyz&abc=123'` is parsed into
```
{
  foo: 'bar',
  abc: ['xyz', '123']
}
```
according to https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options

Reply:
Thanks, pushed a fix!